src/NServiceBus.Core/Sagas/CustomFinderAdapter.cs

```diff
@@ -1,8 +1,7 @@
-namespace NServiceBus.Sagas
+namespace NServiceBus.Saga
```

Comment:
Since it's not relevant to end users don't we want to keep it out of the
root namespace?

On Thu, Mar 26, 2015 at 11:20 AM, Simon Cropp <notifications@github.com>
wrote:

> In src/NServiceBus.Core/Sagas/CustomFinderAdapter.cs
> <https://github.com/Particular/NServiceBus/pull/2693#discussion_r27201036>
> :
>
> > @@ -1,8 +1,7 @@
> > -namespace NServiceBus.Sagas
> > +namespace NServiceBus.Saga
>
> is there really any value in obscuring this api under a nested namespace?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/2693/files#r27201036>.
>